### PR TITLE
Erin/styling errors 2

### PR DESCRIPTION
**Pull request by erinysong**
## Fall 2019 Final Hackathon Loose Ends

- fixed inconsistent headers (casenote and paperwork headers now match)
- paperwork modal styling matches figma
- styled pop up alert for Google OAuth (working on making it fade after 3s but will update as soon as I get that done)

### Related PRs

Remaining style changes (button styling, status bubble colors) are in my other PR for styling from December

### Migrations

Optional - if you added anything to the database through migration(s)

### Tests Performed, Edge Cases

Lol we'll see if we test

### Screenshots

Add screenshots!!! If you'd like

CC: @adowski

### CODE REVIEW

File: app/javascript/components/CaseNoteForm.jsx [hunk 1]
```diff
@@ -129,6 +129,7 @@ class CaseNoteForm extends React.Component {
 
   handleInternalChange = name => event => {
     this.setState({ [name]: !this.state.internal });
+    console.log({ [name]:this.state.internal });
```
Comment (wilsonw926) [inline app/javascript/components/CaseNoteForm.jsx]: remove console.log
